confignode/src/assembly/resources/conf/iotdb-confignode.properties

```diff
@@ -81,6 +81,11 @@ target_config_nodes=0.0.0.0:22277
 # Datatype: Double
 # data_region_per_processor=0.5
 
+# Region allocate strategy
+# Value desc: GREEDY(default)-suitable for small cluster; COPY_SET-suitable for large clusters
```

Comment:
Split this line into two, like the series_partition_executor_class parameter

Reply:
done